recipes/deepstats/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: deepstats
+  version: '0.2'
+
+source:
+  url: https://github.com/gtrichard/deepStats/archive/v0.2.tar.gz
+  sha256: 8996946075379668898c8cf4306502459908f245dcb74db9c98ed786467aa13c
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  host:
```

Comment:
Since you are just copying scripts/binaries in `build.sh`, you can completely remove the `host` section here, and just leave `run`.